runtime/calamari/src/lib.rs

```diff
@@ -484,15 +485,39 @@ parameter_types! {
 	pub const MaxScheduledPerBlock: u32 = 50;
 }
 
+type ScheduleOrigin = EnsureRoot<AccountId>;
+/// Used the compare the privilege of an origin inside the scheduler.
+pub struct OriginPrivilegeCmp;
```

Comment:
The idea is to make sure that a council origin with more yes votes has higher privileges than a council origin with less yes votes. 
Apparently there was an incident in Kusama where a proposal wasn't able to be canceled because of less votes compared to when it originally passed.
Related to https://github.com/paritytech/polkadot/pull/4166